### PR TITLE
CSI-1860 bump base image version

### PR DESCRIPTION
**Pull request by oriyarde**
similar to https://github.com/IBM/ibm-block-csi-operator/pull/68

### CODE REVIEW

File: build/Dockerfile.operator [hunk 1]
```diff
@@ -24,7 +24,7 @@ RUN CGO_ENABLED=1 GOOS=linux go build \
 	github.com/IBM/ibm-block-csi-operator/cmd/manager
 
 
-FROM registry.access.redhat.com/ubi7/ubi-minimal:7.8-237
+FROM registry.access.redhat.com/ubi7/ubi-minimal:7.9-224
```
Comment (ArbelNathan) [inline build/Dockerfile.operator:27]: why not `latest`?

Comment (oriyarde) [reply build/Dockerfile.operator:27]: `latest` is not deterministic.
if you find any `latest` in our projects, please let me know

**Review by barakda — APPROVED** [review]

File: build/Dockerfile.operator (hunk 1 above)
Comment (ArbelNathan) [reply build/Dockerfile.operator:27]: what do you mean by "not deterministic"?

Comment (oriyarde) [reply build/Dockerfile.operator:27]: with `latest`, building the same commit at different times could yield different outputs - which is bad for CI stability

**Review by ArbelNathan — APPROVED** [review]

**Pull request merged**